selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/requestfactory/CheAdminHttpJsonRequestFactory.java

```diff
@@ -10,34 +10,18 @@
  */
 package org.eclipse.che.selenium.core.requestfactory;
 
-import static java.lang.String.format;
-
 import com.google.inject.name.Named;
 import javax.inject.Inject;
 import org.eclipse.che.selenium.core.client.TestAuthServiceClient;
 
 /** @author Dmytro Nochevnov */
-public class TestAdminHttpJsonRequestFactory extends TestHttpJsonRequestFactory {
-
-  private final String adminEmail;
-  private final String adminPassword;
-  private final TestAuthServiceClient authServiceClient;
+public class CheAdminHttpJsonRequestFactory extends TestUserHttpJsonRequestFactory {
```

Comment:
Usually we add **Test** word into the name of selenium test framework classes to avoid confusion with Che product codebase.